tests/integration_tests/build/test_doctests.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""A test that ensures that all doctests pass at integration time."""
+
+import host_tools.cargo_build as host  # pylint:disable=import-error
+
+
+def test_doctests(test_session_root_path):
+    """Run doc tests for all supported targets."""
+    extra_env = ''
+    extra_args = "--target x86_64-unknown-linux-gnu --doc --exclude cpuid"
```

Comment:
Is this test only run on aarch64? If yes, why don't we have a platform config before it? If no, why do you skip the cpuid? The cpuid doc tests need to be run for x86_64.

Reply:
I believe this should only run on `x86_64` since AFAIK doc-tests are run as part of `cargo test` which is only skipped on `x86_64`.

Indeed, `cpuid` should not be skipped.